tests/federation_redaction_test.go

```diff
@@ -0,0 +1,87 @@
+package tests
+
+import (
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/docker"
+	"github.com/matrix-org/complement/internal/federation"
+	"github.com/matrix-org/gomatrixserverlib"
+	"os"
+	"testing"
+	"time"
+)
+
+// test that a redaction is sent out over federation even if we don't have the original event
+func TestFederationRedactSendsWithoutEvent(t *testing.T) {
+	os.Setenv("COMPLEMENT_DEBUG", "1")
+	deployment := Deploy(t, b.BlueprintAlice)
+	defer deployment.Destroy(t)
+
+	roxy := deployment.RegisterUser(t, "hs1", "roxy", "verygoodpass", true)
```

Comment:
Any particular reason for using roxy over Alice from the blueprint?

Reply:
I think this I wanted to have an admin account to make it easier to do redactions but ended up taking care of that in different way. I'll revert to the vanilla alice user.